airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/ConfigRepository.java

```diff
@@ -1517,6 +1517,30 @@ public Geography getGeographyForConnection(final UUID connectionId) throws IOExc
         .fetchOneInto(Geography.class);
   }
 
+  /**
+   * Specialized query for efficiently determining eligibility for the Free Connector Program. If a
```

Comment:
Side thought I want to confirm:

If a workspace user signs up for the free connector program and also has connections with both GA source and destination, we should only consider connections with alpha/beta sources/destinations as free yes?

If a user does not sign up, but already has payment information with us, are they automatically enrolled in the free program?

We might have covered this already. So sorry if I missed it. Checking my understanding.

Reply:
> If a workspace user signs up for the free connector program and also has connections with both GA source and destination, we should only consider connections with alpha/beta sources/destinations as free yes?

Yes, the credit processing cron will only set free_usage: true on Orb events for jobs that contain an alpha/beta connector.

> If a user does not sign up, but already has payment information with us, are they automatically enrolled in the free program?

They are not. Currently our Stripe terms of use for credit payments only allow a single charge to complete that one purchase. So to enroll for the program, they need to explicitly step through the 'setup' mode Stripe checkout which contains a broader agreement for allowing Airbyte to charge that payment in the future